pyecharts/conf.py

```diff
@@ -40,20 +34,27 @@ def jshost(self, jshost):
         self._jshost = remove_trailing_slashes(jshost)
 
     def get_js_library(self, pinyin):
-        for extension in JS_EXTENSIONS:
+        for extension in EXTENSION_MANAGER.get_all_plugins():
             library = extension.get_js_library(pinyin)
             if library is not None:
                 return library
         return None
 
     def chinese_to_pinyin(self, chinese):
-        return CITY_NAME_PINYIN_MAP.get(chinese, chinese)
+        for extension in EXTENSION_MANAGER.get_all_plugins():
+            __PINYIN_MAP__ = extension.registry.get(REGISTRY_PINYIN_MAP, {})
```

Comment:
Make these lines as a method of `JsExtension`, So this module will not access `REGISTRY_PINYIN_MAP` ,even `extension.registry`

Reply:
good point.